apps/website/src/components/content/SubNav.tsx

```diff
@@ -0,0 +1,29 @@
+import IconChevronRight from "@/icons/IconChevronRight";
+import Link from "./Link";
+
+type SubNavProps = {
+  links: {
+    name: string;
+    href: string;
+  }[];
+};
+
+const SubNav = ({ links }: SubNavProps) => (
+  <nav className="sticky left-0 top-0 z-20 mt-0 bg-alveus-green-100/50 text-xl font-bold shadow-md backdrop-blur-2xl">
```

Comment:
Instead of hard-coding the `z-20` here, can we accept `className` as an optional string prop, and use the `classes` util to include it here?